kion/resource_account.go

```diff
@@ -361,6 +362,23 @@ func convertCacheAccountToProjectAccount(c *hc.Client, accountCacheId, newProjec
 	return resp.RecordID, nil
 }
 
+func retryConvertCacheAccountToProjectAccountForAWS(c *hc.Client, accountCacheId, projectId int, startDatecode string, retries int, delay time.Duration) (int, error) {
+	var lastErr error
+	for i := 0; i < retries; i++ {
+		id, err := convertCacheAccountToProjectAccount(c, accountCacheId, projectId, startDatecode)
+		if err == nil {
+			return id, nil
+		}
+		if strings.Contains(err.Error(), "Rule is already in progress") && i < retries-1 {
+			time.Sleep(delay)
+			continue
+		}
+		lastErr = err
+		break
+	}
+	return 0, lastErr
+}
+
```

Comment:
Since this is unique to AWS, it probably should go in the resource_aws_account.go file